docs/0.4.1-alpha/getting_started/installation.md

```diff
@@ -27,6 +27,12 @@ bash -- <(curl -s "https://raw.githubusercontent.com/amber-lang/amber/master/set
 [![Packaging status](https://repology.org/badge/vertical-allrepos/amber-lang.svg)](https://repology.org/project/amber-lang/versions)
 </div>
 
+## Homebrew
+
+```bash
+brew install --HEAD amber-lang/amber/amber-lang
```

Comment:
This directory is called `0.4.1` but we're actually working on `0.5.0` which will probably be the latest release. It will be renamed soon. Would we need `--HEAD` then?